procedures/katello/release.md.erb

```diff
@@ -24,9 +24,10 @@
 
 ## Release Packager
 
-- [ ] Update `katello`, `katello-repos` and `rubygem-katello` in [foreman-packaging](https://github.com/theforeman/foreman-packaging) [rpm/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_rpm_packaging) branch.
+- [ ] Update `katello`, `katello-repos` and `rubygem-katello` in [foreman-packaging](https://github.com/theforeman/foreman-packaging) [rpm/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_rpm_packaging) branch:
+  - [ ] `obal update katello katello-repos rubygem-katello --version <%= version %> <% if is_rc %>--prerelease <%= extra %> --release keep<% end %>`
 - [ ] Merge packaging PR once job is green
-- [ ] Release all packages: `obal release katello katello-repos rubygem-katello`
+- [ ] Wait for [Jenkins to build the packages](https://ci.theforeman.org/job/foreman-packaging-rpm-<%= short_version %>-release/)
```

Comment:
this will generate a bad link, as short_version is Katellos (so 4.1), not Foremans, but I think we don't have a var for that?

Reply:
No, I don't think we do have a variable for that. Perhaps it should have something like `(replace <%= short_version %> with Foreman's version)` so it's at least clear it's not expected to work?

Reply:
Yeah, good call.